### DISCUSSION TITLE
Image optimization

### DISCUSSION THREAD

**Discussion by Nicolasvegam** [vercel/community#7411] [Help] [answered]

**Comment by leerob** [comment, marked as answer]
Hey there, here are some suggestions for optimizing your usage:

- It appears you are using more than the included Pro plan usage of Fast Origin Transfer last month. This is the data transfer between a Vercel Edge Network region and some compute on Vercel (like a Function or Middleware). Here are some options you can explore for [investigating your usage and then optimizing](https://vercel.com/docs/pricing/networking#optimizing-fast-origin-transfer).
- It also appears you're serving some larger images (and GIFs), which are incurring Fast Data Transfer. For non-critical, below the fold assets, you can consider moving these to [Vercel Blob storage](https://vercel.com/docs/storage/vercel-blob), since it appears the majority of your Fast Data Transfer is coming from these assets. You can think about this like "normal data transfer" versus Fast Data Transfer.

These changes should significantly decrease your next bill.

**Reply by Nicolasvegam** [reply]
But what explains the x100 increase from one month to the next one?

**Reply by Nicolasvegam** [reply]
It's really frustrating if you marked as answered but the answer is "we will charge you like crazy for no reason, optimize your stuff"

**Reply by Cyral** [reply]
You do need to optimize your stuff, you are loading 130MB on page load. That is insane. Almost nothing that is being loaded is even displayed. Are you importing images with something like import img as './whatever.png' instead of loading them in HTML _as needed_?

**Reply by Nicolasvegam** [reply]
I need to, that not something new to me dude... There's not need to highlight to feel yourself as pro ;)

But what explains the x100? wtf, I'm a paid customer it's my right to know :S

**Reply by tgdn** [reply]
Your images weigh about 5-10MB each, you have a surge in trafic, and a surge in data transfer.

Reduce the size of all your images and the cost will disappear as fast as it came. I'm not sure why you need to download 10000x10000 images at all, the standard desktop resolution is 1920x1080!

You can convert https://carvuk.com/icons/splash.png to an SVG. The other images can probably be reduced in size (nobody needs 4096x4096 images loaded for the images to look HD). Reduce their dimensions (500x500 is probably enough) and compress them, use a different format (WEBP/JPEG).

**Reply by Dino-Kupinic** [reply]
![image](https://github.com/vercel/community/assets/87943435/6ce73071-7d9f-4093-a56b-de83d067a583)

This image should help (https://www.carvuk.com/) homepage

[a comment by an automated account is not shown]
